kubewatch/pkg/informer/K8sInformer.go

```diff
@@ -666,3 +692,70 @@ func isResourceNotFoundErr(err error) bool {
 	}
 	return false
 }
+
+func getFailedReasonFromPodConditions(conditions []coreV1.PodCondition) string {
+	if len(conditions) == 0 {
+		return "failed"
+	}
+
+	return conditions[0].Message
+}
+
+func significantPodChange(from *coreV1.Pod, to *coreV1.Pod) bool {
```

Comment:
nomenclature should be specific

Reply:
renamed the method to `anychangeFoundInPodStatus`